prune.go

```diff
@@ -100,6 +100,41 @@ func shouldPrune(oldCount, newCount int, pruneOptions PruneOptions) (bool, strin
 	return false, "transactions have not grown significantly"
 }
 
+// checkMongoSupportsOut verifies that Mongo supports "$out" in an aggregation
+// pipeline. This was introduced in Mongo 2.6
+// https://docs.mongodb.com/manual/reference/operator/aggregation/out/
+func checkMongoSupportsOut(db *mgo.Database) bool {
+	var dbInfo struct {
+		VersionArray []int `bson:"versionArray"`
+	}
+	if err := db.Run(bson.M{"buildInfo": 1}, &dbInfo); err != nil {
+		return false
+	}
+	logger.Debugf("buildInfo reported: %v", dbInfo.VersionArray)
```

Comment:
I feel like debug is appropriate. If you are running at debug level you want to find out things like this to understand why it is doing X vs Y. It shouldn't be logged by default, and when it does run, it only logs 1 time per prune run. (so 1 line per 2hrs).
That feels like Debug to me.

Trace would be if it logged 100s/1000s of lines.